server/app.js

```diff
@@ -7,12 +7,11 @@ var path = require('path');
 app.use(bodyParser.json());
 app.use(cors());
 
-app.use(express.static('/client'));
 
+app.use(express.static(path.join(__dirname, "client")));
+app.use(express.static(path.join(__dirname, "../client")));
 
-
-
-app.get('/', (req, res) => res.sendFile("/Users/harrymoore/Projects2022/FP_GooglePagetwo/client/index.html"));
+app.get('/', (req, res) => res.sendFile("/client/index.html", {root: '..'}));
 
 app.get('/search', async (req, res) =>{
     try{
```

Comment:
Interesting way to work through the API, I think it's potentially more steps than you'd need if you try to use some array methods you might get around splitting your description, also potentially looking at :searchQuery and using res.params so your search terms come from the url. Looks like it works which is the main thing!